views/js/pciCreator/ims/likertScoreInteraction/creator/widget/states/Answer.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'taoQtiItem/qtiCreator/widgets/states/factory',
+    'taoQtiItem/qtiCreator/widgets/interactions/states/Answer',
+    'taoQtiItem/qtiCreator/widgets/interactions/helpers/answerState',
+    'taoQtiItem/qtiCreator/widgets/helpers/content'
+], function(stateFactory, Answer, answerStateHelper){
+    'use strict';
+
+    var InteractionStateAnswer = stateFactory.extend(Answer, function initAnswerState(){
+        this.widget.$original.find('.likert input').prop('disabled', true);
+    }, function exitAnswerState(){
+        this.widget.$original.find('.likert input').prop('disabled', false);
+    });
+
+    InteractionStateAnswer.prototype.initResponseForm = function initResponseForm(){
+
+        answerStateHelper.initResponseForm(this.widget, {rpTemplates : ['CUSTOM', 'NONE']});
+    };
```

Comment:
**thought:** Based on whether or not we remove the response processing disabling, this would either change or be removed...

Reply:
I'm leaving it as it is. I was confused for a long time about why `CUSTOM` doesn't appear in the UI until xmlEditRp is installed :(